persistence-cassandra/core/src/main/resources/play/reference-overrides.conf

```diff
@@ -41,13 +41,23 @@ cassandra-journal {
   cluster-id = ${?CASSANDRA_SERVICE_NAME}
 
   plugin-dispatcher = "lagom.persistence.dispatcher"
+
+  # Disable meta columns in the schema to avoid an error at startup when table auto-creation is enabled:
+  # "com.datastax.driver.core.exceptions.InvalidQueryException: Undefined column name meta_ser_id"
+  #
+  # Lagom does not make use of the new metadata feature, so the new schema is not required.
+  #
+  # See below for more details:
+  # - https://github.com/akka/akka-persistence-cassandra#migrations-from-054-to-059
+  # - https://github.com/akka/akka-persistence-cassandra/blob/v0.59/core/src/main/resources/reference.conf#L165-L172
+  meta-in-events-by-tag-view = off
```

Comment:
This is the only real change in this file. All of the other diffs are just automatic trimming of trailing whitespace.